terraform/deployments/govuk-publishing-platform/defaults.tf

```diff
@@ -38,7 +38,7 @@ locals {
     website_root            = local.public_entry_url,
 
     virtual_service_backends = [
-      module.statsd.virtual_service_name
+      module.statsd.virtual_service_names[0]
```

Comment:
Just fyi this is an example of unsafe use of `count`. No action needed here, because there's not much you can do about it until we fix the module, but wanted to raise it anyway since it's a good example of a smell that we should be looking out for in future. This is the card for fixing the issue: https://trello.com/c/iIsJYdHs/227-fix-unsafe-use-of-terraform-count-for-non-identical-objects

Reply:
Thanks. Yes I'd like to remove the container services input - it's great this is already in a card.